### PR TITLE
Fix build fail

### PR DESCRIPTION
**Pull request by Jolyon-S**
**Goals (and why)**:
Failing builds

**Implementation Description (bullets)**:

**Testing (What was existing testing like?  What have you done to improve it?)**:

**Concerns (what feedback would you like?)**:

**Where should we start reviewing?**:

**Priority (whenever / two weeks / yesterday)**:
now 🔥

### CODE REVIEW

File: atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryDbTimeLockFactory.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public KeyValueService createRawKeyValueService(
     public ManagedTimestampService createManagedTimestampService(
             KeyValueService rawKvs, DbTimestampCreationSetting dbTimestampCreationSetting, boolean initializeAsync) {
         return services.computeIfAbsent(
-                dbTimestampCreationSetting.timestampSeries(), (unused) -> new InMemoryTimestampService());
+                dbTimestampCreationSetting.timestampSeries(), unused -> new InMemoryTimestampService());
     }
```
Comment (sudiksha27) [inline atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryDbTimeLockFactory.java:55]: may have to underscore it? but if you've run the build  this is fine.

**Review by sudiksha27 — APPROVED** [review]
👍  Merge when green.

File: atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryDbTimeLockFactory.java (hunk 1 above)
Comment (Jolyon-S) [reply atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/memory/InMemoryDbTimeLockFactory.java:55]: I _think_ that if you call it "unused" it doesn't mind, but we'll see!

**Review by jeremyk-91 — APPROVED** [review]
👍 DPR changes were written in a pairing session.

**Pull request merged**